client/src/styles/main.less

```diff
@@ -18,4 +20,30 @@
   flex-direction: row;
 }
 
-.main { flex: 1; }
+.main {
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  min-height: 100vh;
+}
+
+button {
+  box-shadow: none;
+  -webkit-box-shadow: none;
+  -moz-box-shadow: none;
+  border-radius: 3px;
+  background-color: #00a65a;
+  border: 1px solid #008d4c;
+  color: white;
+  font-weight: 100;
```

Comment:
I'd be quite careful with thin lines and fonts because of non-retina displays usually render them unreadable/invisible.